verification/curator-service/ui/cypress/integration/components/AppTest.spec.ts

```diff
@@ -21,6 +21,24 @@ describe('App', function () {
 
         cy.contains('www.variantb1351.com');
     });
+
+
+    it('allows the user to search by nationality', function () {
```

Comment:
I think it might be a better idea to test this behaviour  by opening filters modal and searching by nationality there instead of SearchBar as this functionality will soon be disabled

Reply:
fixed